boards/ek-lm4f120xl/board.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2015 Rakendra Thapa <rakendrathapa@gmail.com
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_ek-lm4f120xl
+ * @{
+ *
+ * @file        board.c
+ * @brief       Board specific implementations for the Stellaris Launchpad LM4F120 board
+ *
+ * @author      Rakendra Thapa <rakendrathapa@gmail.com>
+ *
+ * @}
+ */
+
+#include "board.h"
+static void leds_init(void);
+
+void board_init(void)
+{
+    /* initialize the boards LEDs, this is done for debugging purpose */
+    leds_init();
+
+    /* initialize the CPU */
+    cpu_init();
+}
+
+/**
+ * @brief initialize the boards on-boards LEDS.
+ */
+static void leds_init(void)
+{
+    // enable clock for PORTF
```

Comment:
Is it necessary to include c-style comments ??
Since in coding convention its already mentioned that it should be c99 compliant shouldn't this be okay. 
Also stellarisware header files have all the comments and copyright section with // style comments. Though just comments, I still didn’t want to touch anything in these files and use them as provided by TI.